src/client/main.tsx

```diff
@@ -29,6 +49,8 @@ export const Main = (props: ClientState) => {
         `}
       />
       <ClientStateProvider clientState={props}>
+        {/* Uncomment below line for AB Test Demo */}
```

Comment:
Should this comment be there ? It might just be worth adding these instructions to the readme or `ABTestDemo.tsx`